runtime/crab/src/pallets/timestamp.rs

```diff
@@ -20,7 +20,7 @@
 use crate::*;
 
 impl pallet_timestamp::Config for Runtime {
-	type MinimumPeriod = ();
+	type MinimumPeriod = ConstU64<{ SLOT_DURATION / 2 }>;
```

Comment:
![image](https://github.com/user-attachments/assets/10b968a3-5261-49d5-83f3-3b9fd52ce4d1)
According to the documentation, the `MinimumPeriod` needs to be adjusted in Phase 3, along with the `AllowMultipleBlocksPerSlot`. Therefore, I suggest reverting this modification for the Crab and Darwinia runtimes.

Reply:
It's the revert.

Reply:
Okay. https://github.com/darwinia-network/darwinia/blob/v6.6.3/runtime/crab/src/pallets/timestamp.rs I checked the original config.